openshift/s2i/bin/run

```diff
@@ -97,7 +98,7 @@ if is_gunicorn_installed; then
     export WEB_CONCURRENCY=${WEB_CONCURRENCY:-$(get_default_web_concurrency)}
 
     echo "---> Serving application with gunicorn ($APP_MODULE) ..."
-    exec gunicorn "$APP_MODULE" --chdir=$APP_HOME --bind=0.0.0.0:8080 --config "$APP_CONFIG" --preload
+    exec gunicorn "$APP_MODULE" --chdir=$APP_HOME --bind=0.0.0.0:8000 --config "$APP_CONFIG" --preload
```

Comment:
These port changes I assume will mean we'll need to make some adjustments to our CI/QA config?

Reply:
Actually, there is a little more work to do on these, these port changes should actually be driven by the `cdappconfig.json` file which is mounted inside the container. Usually apps will run up a service and query the JSON document, failing back on an env var if clowder isn't enabled. We can emulate the same functionality here if you wish?

Reply:
That sounds like it would be a great approach. 👍